pkg/jwx/jwx.go

```diff
@@ -84,13 +84,30 @@ func DecodeAccessToken(accessToken string, e, n *string) (*jwt.Token, *jwt.MapCl
 	}
 
 	claims := &jwt.MapClaims{}
+
+	if expectedAudience != "" {
```

Comment:
a lot of duplicated code, let's change it to:
```go
audValidation := jwt.WithoutAudienceValidation()
if expectedAudience != "" {
    audValidation = jwt.WithAudience(expectedAudience)
} 
....
```

Reply:
good call